src/components/RightPane/RightPaneStore.ts

```diff
@@ -29,13 +29,25 @@ class RightPaneStore extends EventEmitter {
     private activeTab: number;
     private doDisplaySearch: boolean;
     private openSpotAlertPopoverActive: boolean;
+    private urlCourseCodeValue: string;
+    private urlTermValue: string;
+    private urlGEValue: string;
+    private urlCourseNumValue: string;
+    private urlDeptLabel: string;
+    private urlDeptValue: string;
     constructor() {
         super();
         this.setMaxListeners(15);
         this.formData = { ...defaultFormValues }; // creates shallow copy
```

Comment:
```suggestion
        this.formData = structuredClone(defaultFormValues)
```
This is a built-in way to create a deep copy. I think we had issues with `structuredClone` on older browsers before tho.